.github/workflows/sdk.yaml

```
@@ -372,8 +368,7 @@ jobs:
           sudo unzip -x /tmp/protoc.zip
           protoc --version
       - name: Regenerate all the code
-        working-directory: generator
-        run: go run ./cmd/sidekick refreshall -project-root ..
+        run: go run github.com/googleapis/librarian/cmd/sidekick@v0.1.1-0.20250804160504-ab9b4cc99e9a refreshall -project-root ..
```

The `project-root` flag needs to change:

```suggestion
        run: go run github.com/googleapis/librarian/cmd/sidekick@v0.1.1-0.20250804160504-ab9b4cc99e9a refreshall -project-root .
```

---

Fixed... 🤞